index.js

```diff
@@ -3,32 +3,49 @@ const { platform, arch } = process
 /** @typedef {typeof import('./client.d')} Client */
 /** @type {Client} */
 let nativeBinding = undefined
-
-if (platform === 'win32' && arch === 'x64') {
-    nativeBinding = require('./dist/win64/steamworksjs.win32-x64-msvc.node')
-} else if (platform === 'linux' && arch === 'x64') {
-    nativeBinding = require('./dist/linux64/steamworksjs.linux-x64-gnu.node')
-} else if (platform === 'darwin') {
-    if (arch === 'x64') {
-        nativeBinding = require('./dist/osx/steamworksjs.darwin-x64.node')
-    } else if (arch === 'arm64') {
-        nativeBinding = require('./dist/osx/steamworksjs.darwin-arm64.node')
+if(process.env.IS_TESTING){
+    if (platform === 'win32' && arch === 'x64') {
+      nativeBinding = require('./dist/win64/steamworksjs.win32-x64-msvc.node')
+    } else if (platform === 'linux' && arch === 'x64') {
+        nativeBinding = require('./dist/linux64/steamworksjs.linux-x64-gnu.node')
+    } else if (platform === 'darwin') {
+        if (arch === 'x64') {
+            nativeBinding = require('./dist/osx/steamworksjs.darwin-x64.node')
+        } else if (arch === 'arm64') {
+            nativeBinding = require('./dist/osx/steamworksjs.darwin-arm64.node')
+        }
+    } else {
+        throw new Error(`Unsupported OS: ${platform}, architecture: ${arch}`)
     }
 } else {
-    throw new Error(`Unsupported OS: ${platform}, architecture: ${arch}`)
+  if (platform === 'win32' && arch === 'x64') {
+      nativeBinding = require('steamworks.js/dist/win64/steamworksjs.win32-x64-msvc.node')
+  } else if (platform === 'linux' && arch === 'x64') {
+      nativeBinding = require('steamworks.js/dist/linux64/steamworksjs.linux-x64-gnu.node')
+  } else if (platform === 'darwin') {
+      if (arch === 'x64') {
+          nativeBinding = require('steamworks.js/dist/osx/steamworksjs.darwin-x64.node')
+      } else if (arch === 'arm64') {
+          nativeBinding = require('steamworks.js/dist/osx/steamworksjs.darwin-arm64.node')
+      }
+  } else {
+      throw new Error(`Unsupported OS: ${platform}, architecture: ${arch}`)
+  }
 }
 
+
 let runCallbacksInterval = undefined
 
 /**
  * Initialize the steam client or throw an error if it fails
  * @param {number} [appId] - App ID of the game to load, if undefined, will search for a steam_appid.txt file
+ * @param {boolean} [networking] - Enable networking
  * @returns {Omit<Client, 'init' | 'runCallbacks'>}
 */
-module.exports.init = (appId) => {
+module.exports.init = (appId, networking) => {
```

Comment:
networking?

Reply:
Ahh good catch. this was not intentional. Forked this from my networking branch.